bundles/core/src/main/java/com/adobe/cq/wcm/core/components/models/Text.java

```diff
@@ -26,7 +26,7 @@
  * @since com.adobe.cq.wcm.core.components.models 11.0.0
  */
 @ConsumerType
-public interface Text extends ComponentExporter {
+public interface Text extends ComponentExporter, DataLayerProperties {
```

Comment:
Each model interface should just have one addional method which will return a DataLayer object with  `getDataLayer()` instead of extending from another interface to follow the composite reuse principle.

In HTL this would then look like `list.dataLayer.String` instead of `list.dataLayerString`

Reply:
Done in 9f036b3